src/examples/midi_test/midi_test.py

```diff
@@ -0,0 +1,19 @@
+import tensorflow as tf
+from ...datasets import preprocess as pre
+
+dataset = tf.data.Dataset.from_tensor_slices(["test.midi"])
+
+dataset = pre.pipeline(
+    [
+        pre.midi(),
+        pre.unbatch(),
+        pre.prefetch(),
+        pre.batch(100),
+        pre.numpy(),
+        pre.decode_midi(),
+    ]
+)(dataset)
```

Comment:
Nitpicking: no need for the extra indentation
```suggestion
dataset = pre.pipeline([
    pre.midi(),
    pre.unbatch(),
    pre.prefetch(),
    pre.batch(100),
    pre.numpy(),
    pre.decode_midi(),
])(dataset)
```

Reply:
The extra indents were auto added by python black.